src/api/index.js

```diff
@@ -273,11 +273,6 @@ export default new ApolloServer({
       userByHandle: new UnorderedDataLoader(db.User.findByHandles.bind(db.User), ({ handle }) => handle, user => new User(user)),
       upcomingStayByUserId: new UnorderedDataLoader(db.Stay.findUpcomingStays.bind(db.Stay), ({ userId }) => userId, stay => new Stay(stay)),
       currentOrLastStayByUserId: new UnorderedDataLoader(db.Stay.findCurrentOrLastStays.bind(db.Stay), ({ userId }) => userId, stay => new Stay(stay)),
-      isHome: new DataLoaderWithNoIdParam(async () => {
```

Comment:
We need to properly remove this;

 1. Camera should always tell Karen about motion (Karen dismisses it).
 2. Remove the code to toggle Synology between Home and Away mode.